.github/workflows/sync_messages_repo.yaml

```diff
@@ -0,0 +1,76 @@
+name: Sync i18n to civiform-messages repo
+
+permissions:
+  contents: read # Only used for civiform/civiform
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+    paths:
+      - 'server/conf/i18n/**'
+  schedule:
+    # Run daily at 2 AM UTC
+    - cron: '0 2 * * *'
+
+jobs:
+  sync-i18n:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout civiform repo
+        uses: actions/checkout@v4
+        with:
+          path: civiform
+
+      - name: Checkout civiform-messages repo
+        uses: actions/checkout@v4
+        with:
+          repository: civiform/civiform-messages
+          token: ${{ secrets.CIVIFORM_AUTOMATION_MESSAGES_WRITE }}
+          path: civiform-messages
+          persist-credentials: true
+
+      - name: Configure git
+        run: |
+          git config --global user.name "civiform-github-automation"
+          git config --global user.email "civiform-dev@exygy.com"
+
+      - name: Sync i18n files
+        run: |
+          rm -f civiform-messages/server/conf/i18n/*
+          if [ -d "civiform/server/conf/i18n" ]; then
+            cp civiform/server/conf/i18n/* civiform-messages/server/conf/i18n/
+          else
+            echo "Warning: server/conf/i18n directory not found in civiform repo"
+            exit 1
+          fi
+
+      - name: Check for changes
+        id: check-changes
+        run: |
+          cd civiform-messages
+          git add -A
+          
+          # Check if there are any changes
+          if git diff --staged --quiet; then
+            echo "No changes detected"
+            echo "has-changes=false" >> $GITHUB_OUTPUT
+          else
+            echo "Changes detected:"
+            git diff --staged --stat
+            echo "has-changes=true" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Commit and push changes
+        if: steps.check-changes.outputs.has-changes == 'true'
+        run: |
+          cd civiform-messages
+          
+          # Get the latest commit hash from civiform repo for reference
+          CIVIFORM_COMMIT=$(cd ../civiform && git rev-parse HEAD)
+          
+          git commit -m "Sync i18n files from civiform@${CIVIFORM_COMMIT}
```

Comment:
The link created in the first line 404s 

https://github.com/civiform/civiform-messages/commit/92c013f302d75b019d6a95f1c1dca0790a1a0d9a